### ISSUE TITLE
Bats from Mount Whitney

### ISSUE THREAD

**Issue by pipfrosch** [pipfrosch/JoMV001#2]
1. In Abstract, after "Mrs. Hilda W. Grinnell" reference, period is missing.
2. Add [] around authority for Nomenclature Authority
3. Missing semicolon after `41` in White-edged bat Measurements

**Comment by pipfrosch** [comment]
'Nother issue before can be closed - id `bmw02.msm` should be in spans with with `role={list,item}`
